src/main/java/com/elmsoftware/env/EnvironmentSettingsModule.java

```diff
@@ -44,15 +56,31 @@ public void configure(final Binder binder) {
 
 		log.debug("Loading environment {} using resource {}", environment, resourceName);
 		final EnvironmentSettings settings = EnvironmentSettings.load(resourceName);
-		final Map<String, String> properties = settings.merge(environment, settingProvider);
+		final Properties properties = new Properties();
+		properties.putAll(settings.merge(environment, settingProvider));
+		settingPostProcessor.process(properties);
 
 		Names.bindProperties(binder, properties);
 		configure(binder, properties);
+	}
 
+	protected void configure(final Binder binder, final Properties properties) {
```

Comment:
the move from a map to properties makes sense to me. I deprecated the map implementation so we can feel okay with removing it on a later release, but left a default implementation on the properties version so there's no api change with this.